lib/chewy/type/crutch.rb

```diff
@@ -23,7 +24,11 @@ def #{name}
 
       module ClassMethods
         def crutch name, &block
-          self._crutches = _crutches.merge(name.to_sym => block)
+          if block
+            self._crutches = _crutches.merge(name.to_sym => block)
+          else
+            self._crutches_classes = _crutches_classes + [name]
```

Comment:
@DNNX Had a better idea than that I suggested (with `crutch: 'name#name'`). Let's wait for his input here.